resources/configure-metrics/trust-policy.json

```diff
@@ -0,0 +1,17 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "Federated": "<CHANGE_ME>"
+            },
+            "Action": "sts:AssumeRoleWithWebIdentity",
+            "Condition": {
+                "StringEquals": {
+                    "<CHANGE_ME>:sub": "system:serviceaccount:<CHANGE_ME>:<CHANGE_ME>"
```

Comment:
Maybe you can add the project and as name so that students will only have to change the OIDC as they did in for the logs exercise.

Reply:
Sure. Done